Proposal-Process.md

```diff
@@ -0,0 +1,14 @@
+For a proposal to create or change a process foundation-wide, a PR should be
```

Comment:
Works for me. I'm inclined to specify Working Groups too because (if I understand correctly) not all Working Groups need to fall under a particular Top Level Project.